src/grt/README.md

```diff
@@ -87,225 +122,337 @@ used to trade-off wirelength and path depth.  Alpha is between 0.0
 
+```
 [WARNING GRT-0243] Unable to repair antennas on net with diodes.
-
 ```
+
+### Write Global Routing Guides
+
+```tcl
 write_guides file_name
 ```
-The `write_guides` generates the guide file from the routing results.
+
+| Switch Name | Description |
+| ----- | ----- |
+| `file_name` | Guide file name. |
 
 Example: `write_guides route.guide`.
 
+### Estimate Global Routing Parasitics
+
 To estimate RC parasitics based on global route results, use the `-global_routing`
 option of the `estimate_parasitics` command.
 
+```{note}
+To see the function definition for `estimate_parasitics`, refer to 
+[Resizer docs](../rsz/README.md#estimate-parasitics).
 ```
+
+```tcl
 estimate_parasitics -global_routing
 ```
 
-```
-draw_route_guides net_names [-show_pin_locations]
-```
+### Plot Global Routing Guides
+
 The `draw_route_guides` command plots the route guides for a set of nets.
 To erase the route guides from the GUI, pass an empty list to this command:
 `draw_route_guides {}`.
-The `show_pin_locations` flag draw circles for the pin positions on the routing grid.
-
-## Report wire length
 
-```
-report_wire_length [-net net_list]
-                   [-file file]
-                   [-global_route]
-                   [-detailed_route]
-                   [-verbose]
+```tcl
+draw_route_guides 
+    net_names 
+    [-show_pin_locations]
 ```
 
-Options description:
+#### Options
 
--   `net`: A list of nets. Use `*` to report the wire length for all nets of the design.
--   `file`: The name of the file for the wire length report.
--   `global_route`: This flag is used to report the wire length of the global routing.
--   `detailed_route`: This flag is used to report the wire length of the detailed routing.
--   `verbose`: This flag enables the full reporting of the wire length information.
+| Switch Name | Description |
+| ----- | ----- |
+| `net_names` | Tcl list of set of nets (e.g. {net1, net2}). |
+| `-show_pin_locations` | Draw circles for the pin positions on the routing grid. |
+
+### Report Wirelength
 
 The `report_wire_length` command reports the wire length of the nets. Use the `-global_route`
 and the `-detailed_route` flags to report the wire length from global and detailed routing,
 respectively. If none of these flags are used, the tool will identify the state of the design
 and report the wire length accordingly.
 
+```tcl
+report_wire_length 
+    [-net net_list]
+    [-file file]
+    [-global_route]
+    [-detailed_route]
+    [-verbose]
+```
+
+#### Options
+
+| Switch Name | Description | 
+| ----- | ----- |
+| `net` | A list of nets. Use `*` to report the wire length for all nets of the design. |
+| `file` | The name of the file for the wire length report. |
+| `global_route` | This flag is used to report the wire length of the global routing. |
+| `detailed_route` | This flag is used to report the wire length of the detailed routing. |
+| `verbose` | This flag enables the full reporting of the wire length information. |
```

Comment:
```suggestion
| `-net` | List of nets to report the wirelength. Use `*` to report for all nets of the design. |
| `-file` | The name of the file for the wirelength report. |
| `-global_route` | Report the wirelength of the global routing. |
| `-detailed_route` | Report the wirelength of the detailed routing. |
| `-verbose` | This flag enables the full reporting of the wire length information. |
```

Reply:
@eder-matheus, what does "full reporting" in the `-verbose` actually mean? Which additional info is reported?

Reply:
Here's an example of the report with and without the `-verbose` flag:
<img width="605" alt="image" src="https://github.com/The-OpenROAD-Project/OpenROAD/assets/43450810/b7a6d867-a42b-41b0-abcc-1d166b09e08d">

In summary, the `-verbose` gives details of the wire length per routing layer.

Reply:
@luarss 
Please make this distinction clear on what the `-verbose` flag adds to the report.